frontend/src/App.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import {BrowserRouter, Routes, Route} from 'react-router-dom';
+import MetricCreateEdit from './pages/MetricCreateEdit';
+import Services from './pages/Services.jsx';
+import Metrics from './pages/Metrics';
+import NotFound from './pages/NotFound';
+import Scorecards from './pages/Scorecards';
+
+function App() {
+  return (
+    <BrowserRouter>
```

Comment:
read [this tutorial](https://reactrouter.com/en/main/start/tutorial#adding-a-router) and refactor this to extract the routes definition into a separate file

Reply:
create a folder called routes or router and move the generated file to it.

Reply:
Done